### PR TITLE
OLD -- suspension loop space adjunction

### PR DESCRIPTION
**Pull request by morphismz**
This pr contains partial progess towards constructing the pointed equivalence of the suspension loop space adjunction. The construction of the pointed equivalence is according to the following strategy:
- constructed the main pointed equivalence as the pointed composite of "stepping stone" pointed equivalences
- construct the stepping stone pointed equivalences as lemmas in such a way that they they are reusable
- create definitions to make the code readable

This pr contains:
- the first stepping stone equivalence, named `equiv-pointed-map-pointed-suspension-structure-pointed-function-type`
- related lemmas and definitions

I decided to construct the pointed equivalence this way since proving the previously constructed equivalence is pointed proved to be too hard; it has very bad reduction behavior.

### CODE REVIEW

File: src/synthetic-homotopy-theory/suspensions-of-types.lagda.md [hunk 1]
```diff
@@ -482,7 +486,8 @@ module _
         ( id))
 ```
 
-#### The equivalence between pointed maps out of the suspension of X and pointed maps into the loop space of Y
+#### The equivalence between pointed maps out of the suspension of X and
```
[the remark this answers is not in the document]
Comment (morphismz) [reply src/synthetic-homotopy-theory/suspensions-of-types.lagda.md:489]: yes, your correct. I'll revert that change

**Review by fredrik-bakke — COMMENTED** [review]
Trying out the "suggestion" feature of GitHub, here are some minor comments on formatting ^^

**Comment by fredrik-bakke** [comment]
No worries! Just to check, you could simply commit my suggestions by clicking a button below them, no?

**Comment by EgbertRijke** [comment]
Given that you marked this as old, can we close this PR?

**Pull request closed without merging**